src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -14,11 +14,11 @@ import DiffExprManager from '../../../../components/data-exploration/differentia
 import Embedding from '../../../../components/data-exploration/embedding/Embedding';
 import { COMPONENT_TYPE, HeatmapPlot } from '../../../../components/data-exploration/heatmap/HeatmapPlot';
 import HeatmapSettings from '../../../../components/data-exploration/heatmap/HeatmapSettings';
+import MosaicCloseButton from '../../../../components/MosaicCloseButton';
```

Comment:
New component instead of the `RemoveButton` that relied on some Blueprint.js functionality.